.codecov.yml

```diff
@@ -0,0 +1,12 @@
+codecov:
```

Comment:
```suggestion
# see documentation at ...
codecov:
```

Reply:
it would be always nice to have some initial clue where to look :)

Reply:
done